lib/ansible/cli/galaxy.py

```diff
@@ -1313,7 +1318,8 @@ def execute_list(self):
         if context.CLIARGS['type'] == 'role':
             self.execute_list_role()
         elif context.CLIARGS['type'] == 'collection':
-            self.execute_list_collection()
+            output_format = context.CLIARGS.get('output_format', False)
```

Comment:
This should be moved inside `execute_list_collection()` for consistency — all other code accesses `context.CLIARGS` directly so it's unnecessary to have an injection here.